reco_utils/dataset/criteo_dac.py

```diff
@@ -110,34 +117,64 @@ def _get_schema(include_label=True):
   return schema
 
 
-def _load_datafile(local_cache_path="dac.tar.gz", dbutils=None):
+def criteo_urls(size):
+  """Return the url of criteo based on the input size
+
+  Args:
+    size (str): Size of criteo dataset. It can be "full" or "sample".
+
+  Returns:
+    str: url of Criteo dataset.
+
+  """
+  url = {
+    "full": CRITEO_URL_FULL,
+    "sample": CRITEO_URL_SAMPLE
+  }
+  return url[size]
+
+
+def download_criteo(size="full", filename="dac.tar.gz", work_directory="."):
```

Comment:
FYI @loomlike, please take a look at the way we are splitting the code to follow single responsibility. I think this can be also done in movielens.py